.github/workflows/mkdocs-ghpages.yaml

```diff
@@ -10,5 +10,5 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.x
-      - run: pip install mkdocs-material 
+      - run: pip install mkdocs-material mkdocs-static-i18n
```

Comment:
how about babel?
https://www.mkdocs.org/dev-guide/translations/

Reply:
https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins#i18n
https://github.com/mkdocs/catalog
https://github.com/ultrabug/mkdocs-static-i18n
and the user guide can be found there.

I suppose we just need follow user guide right?
https://www.mkdocs.org/user-guide/localizing-your-theme/
hence... do we need babel?